### ISSUE TITLE
bad function in examples

### ISSUE THREAD

**Issue by tbenst** [uber/pyro#516]
This issue is on http://pyro.ai/examples/intro_part_ii.html, so hopefully this is the right place for an issue.

```python
# equivalent to pyro.condition(scale, data={"measurement": Variable(torch.ones(1))})
def scale_obs(guess):
    z = pyro.sample("weight", dist.normal, guess, Variable(torch.ones(1)))
     # here we attach an observation measurement == 1
    return pyro.sample("measurement", dist.normal, weight, Variable(torch.ones(1)),
                       obs=Variable(torch.Tensor([0.1])))

# equivalent to scale_obs:
def scale_obs(guess):
    z = pyro.sample("weight", dist.normal, guess, Variable(torch.ones(1)))
    # here we attach an observation measurement == 1
    return pyro.observe("measurement", dist.normal, Variable(torch.ones(1)),
                        weight, Variable(torch.Tensor([0.1])))
```

`weight` is not defined, I'm guessing that `z` should be `weight`

**Comment by eb8680** [comment]
> weight is not defined, I'm guessing that z should be weight

Yes, this is correct, I'll fix it.

**Comment by manuelhaussmann** [comment]
That section also contains a couple of mixed up numbers, where the observed measurement is stated to be 0.1 in the main text, while the comments in the code claim `measurement==1`, with the actual code agreeing with the former. One code example is the snipped tbenst posted about the z/weight issue.

**Comment by eb8680** [comment]
Closed by #559
